src/Qir/Tests/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 3.10)
+
+message(INFO "*** build config: ${CMAKE_BUILD_TYPE}")
+
+# set the project name and version
+project(qir-tests)
+
+# specify the C++ standard, compiler and other tools
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -DDEBUG")
+
+if(WIN32)
+    if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_DLL -D_MT -Xclang --dependent-lib=msvcrtd")
+    else()
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_DLL -D_MT -Xclang --dependent-lib=msvcrt")
+    endif()
+endif()
```

Comment:
(Minor)
I have an impression that this is done in more than one place.
Consider extracting the common part into a separate file invoked from other files.

Reply:
That's true, and definitely clean-up we can consider for the future, especially as we grow the number of folders where we do native builds using cmake. I'll hold off on that for now though.